tests/parse/test_parse_case.py

```diff
@@ -62,6 +62,14 @@ def test_parse_case_owner(scout_config):
     assert case_data["owner"] == scout_config["owner"]
 
 
+def test_parse_case_limsid(scout_config):
```

Comment:
Could you add a very short doctring for this function? One line would be sufficient